Minor help additions and corrections; English translation.

MKadaner opened the pull request.

alabuzhev commented on far/FarEng.hlf.m4:
```diff
@@ -6187,16 +6199,37 @@ $ #far:config Viewer.F8CPs#
+
+    This parameter can be changed via ~far:config~@FarConfig@ only.
+
+@Panel.Tree.AutoChangeFolder
```
These parameters are available in the UI, do we need to describe them again? Maybe just add a reference to the Tree Settings topic?

MKadaner replied:
Thank you for pointing this out. I added help topic aliases. Was it what you meant? Or you'd prefer to have dedicated topics containing only "see..." references?

alabuzhev replied:
Please have a look [how Far searches for an appropriate topic](https://github.com/FarGroup/FarManager/blob/5f4eae4156116a0857f27305452c95313fbed007/far/config.cpp#L2287) when you press Shift+F1 in far:config.

For example, for the "Panel.Tree.AutoChangeFolder" key Far first will try a topic of the same name and then "Panel.TreeSettings".

What I mean if a parameter is not a some semi-secret far:config setting and already available in UI and already described in a help topic of that UI - we really don't need to describe it separately: it's harder to maintain and keep in sync two copies and it leads the user to the wrong direction.
Instead it's better to just let Far show the topic of the corresponding UI.

However, it's not always possible: for example, we don't have the "Panel.TreeSettings" topic, it's just "TreeSettings", so it won't be found automatically. Renaming topics to follow the schema might be not a good idea since they have been there for ages and some third party plugins might refer to them in their hlf files, so it's better to add an alias, e.g. `@Panel.TreeSettings=TreeSettings`.

MKadaner replied:
Got it! Thank you. Single alias to the UI page instead of two aliases (which I did in iteration 2) is certainly better. Will change in iteration 3.

alabuzhev commented on far/FarRus.hlf.m4:
```diff
@@ -5519,6 +5520,15 @@ Left/Right и Num4/Num6 действуют по-разному в зависим
     Изменение этого параметра возможно через ~far:config~@FarConfig@
 
+@Panel.Layout.DoubleGlobalColumnSeparator
+$ #far:config Panel.Layout.DoubleGlobalColumnSeparator#
```
Same as in English file here and below - maybe we can reference existing topics?

MKadaner replied:
Ditto.

alabuzhev replied:
Ditto again :)

MKadaner replied:
Sorry for missing this before. Too many minor changes; just lost track of the things. Sure, I will make it an alias (in both files). Thank you for carefully verifying the changes.

MKadaner commented on far/FarEng.hlf.m4:
```diff
@@ -4210,12 +4210,15 @@ be separated with a character "#;#" or "#,#". If the name of a target contains
+panel tree. #Shift+F10# allows to open the tree for the path entered
+in the input line (if several paths are entered, only the first one is
+taken into account). If the option "#Process multiple destinations#"
+is enabled, the dialog selected in the tree is appended to the edit
```
Sure, I'll fix it. I did not translate this paragraph, just split it into two because the topics are unrelated. (Did the same in Russian help.) I was going to rewrite it, but was not sure about the intended behavior of `Shift+F10`. It looks like something does not work around this place. I asked the question at the [forum](https://forum.farmanager.com/viewtopic.php?p=148538#p148538).

MKadaner commented on far/FarEng.hlf.m4:
```diff
@@ -5846,7 +5858,7 @@ $ #far:config System.CopySecurityOptions#
 права всегда копируются.
 
- Изменение этого параметра возможно через ~far:config~@FarConfig@
+    Изменение этого параметра возможно через ~far:config~@FarConfig@
```
This line appears about 60 times in the English help, in all untranslated `far:config` topics. I did not change them to avoid language mix inside single topic. If you think it is still better to change them all even before translating the topics themselves, please let me know. I am not strong on this.

alabuzhev replied:
No need to change all of them.
I mentioned this one only because you already touched it.
It's ok to leave as is for now.

MKadaner replied:
> I mentioned this one only because you already touched it.

I understand, and I actually thought about translating the line when I touched it. However, I finally decided to keep the topics in the single language. I'll keep them in Russian.

MKadaner commented on far/FarEng.hlf.m4:
```diff
@@ -6164,7 +6176,7 @@ $ #far:config History.*#
       время жизни элемента (.Lifetime) = 90 дней
 
-    Изменить эти параметры возможно через ~far:config~@FarConfig@
+    Изменить эти параметры можно через ~far:config~@FarConfig@
```
Ditto.

MKadaner commented:
I addressed all review comments. Also, on the @CopyFiles help page, аккуратно заменил все простые двойные кавычки на типографские -- "ёлочки" в русском и "лапки" в английском тексте.

MKadaner commented:
Please merge the other PR first. Then I'll rebase and republish this one.

alabuzhev commented on far/FarEng.hlf.m4:
```diff
@@ -5371,6 +5386,15 @@ Left/Right и Num4/Num6 действуют по-разному в зависим
 
     Изменение этого параметра возможно через ~far:config~@FarConfig@
 
+@Panel.Layout.DoubleGlobalColumnSeparator
```
Could it be an alias to @PanelSettings topic please?

MKadaner replied:
Sure. Done.

alabuzhev commented on far/FarEng.hlf.m4:
```diff
@@ -2555,15 +2558,15 @@ $ #Settings dialog: panel#
+                          will synchronously change the folder on the other panel.
+                          If turned off, to change the folder from the tree panel,
+                          you need to press #Enter#.
+                          Related ~far:config~@FarConfig@ parameter is #Panel.Tree.AutoChangeFolder#.
```
No need to specify far:config equivalent for every option,

MKadaner replied:
OK. Removed.

alabuzhev commented on far/FarEng.hlf.m4:
```diff
@@ -2555,15 +2558,15 @@ $ #Settings dialog: panel#
-                          #tree3.far#.
+  #Minimum number#          The minimal number of folders on the disk for which
+  #of folders#              folder tree cache file #tree3.far# will be created.
+                          Related ~far:config~@FarConfig@ parameter is #Panel.Tree.MinTreeCount#.
```
No need to specify far:config equivalent for every option,

MKadaner replied:
Removed.

alabuzhev commented on far/FarRus.hlf.m4:
```diff
@@ -2608,15 +2610,15 @@ $ #Настройка панели#
+                          по ~дереву папок~@TreePanel@ будет синхронно менять папку в другой панели.
+                          Если эта опция выключена, то для смены папки из дерева папок
+                          вы должны нажать #Enter#.
+                          Соответствующий параметер ~far:config~@FarConfig@ -- #Panel.Tree.AutoChangeFolder#.
```
No need to specify far:config equivalent for every option,

alabuzhev commented on far/FarRus.hlf.m4:
```diff
@@ -2608,15 +2610,15 @@ $ #Настройка панели#
-                          #tree3.far#.
+  #Минимальное#             Минимальное количество папок на диске, для которого
+  #количество папок#        будет создаваться файл кэша дерева каталогов #tree3.far#.
+                          Соответствующий параметер ~far:config~@FarConfig@ -- #Panel.Tree.MinTreeCount#.
```
No need to specify far:config equivalent for every option,

MKadaner replied:
Removed.

MKadaner commented on far/FarEng.hlf.m4:
```diff
@@ -3,43 +3,44 @@
+@Panel.InfoSettings=InfoPanel
+@Panel.Left=PanelCmdSort
+@Panel.Right=PanelCmdSort
+@Panel.Tree=TreeSettings
```
This should be
`@Panel.TreeSettings=TreeSettings`
But then it resolves the `Panel.Tree.TreeFileAttr` `far:config` parameter to the same "Tree Settings" UI page, where there is no corresponding option exists.

Maybe define two separate aliases, as it was in the previous iteration?
```
Panel.Tree.AutoChangeFolder=TreeSettings
Panel.Tree.MinTreeCount=TreeSettings
```
What do you think? I am going to bed. Until tomorrow.

alabuzhev replied:
Sure, that's fine.
My comment was about avoiding duplicates in the text if possible, not about minimising the number of aliases :)

MKadaner replied:
Thank you. Understood. Done.

MKadaner commented:
Ugh! Let's try again. :)

alabuzhev commented:
Thanks!

MKadaner commented:
Thank you!
